addons/main/functions/fnc_findBuildings.sqf

```diff
@@ -37,10 +37,8 @@ private _housePos = [];
     _housePos append (_house buildingPos -1);
     if (_findDoors) then {
         {
-            if ("door" in toLower (_x)) then {
-                _housePos pushBack (_house modelToWorld (_house selectionPosition _x));
-            };
-        } forEach (selectionNames _house);
+            _housePos pushBack (_house modelToWorld (_house selectionPosition _x));
+        } forEach ((_house selectionNames "viewgeometry") select {"door" in _x});
```

Comment:
I personally don't like that we loop 2x through the items here with first select and then forEach.  we should have the if still in the foreach

Reply:
Re-added the if. Select was faster, but this is definitely easier to read.